### PR TITLE
Make scrolling more generic

### PR DESCRIPTION
**Pull request by ptoffy**
This allows to use the scrolling buttons all over the page rather than just in the showcase

### CODE REVIEW

**Review by 0xTim — CHANGES_REQUESTED** [review]
Lets keep the number of sponsors at 3 and only show the buttons when the page size means they run off the screen

**Comment by ptoffy** [comment]
@0xTim I added sponsors to be able to see how scrolling works when there are more

**Comment by 0xTim** [comment]
Ok we can test the hiding on the main page then. Other option is move the sponsors card to the design page that contains all the elements

**Review by 0xTim — APPROVED** [review]

File: src/scss/vapor/main-site.scss [hunk 1]
```diff
@@ -685,15 +679,34 @@ h2 {
         width: 56px;
         height: 56px;
 
+        &:hover {
+            background: $grey-300;
+        }
+
+        &.disabled {
```
Comment (0xTim) [inline src/scss/vapor/main-site.scss:686]: We need dark mode versions of the disabled state

**Review by 0xTim — CHANGES_REQUESTED** [review]

**Review by 0xTim — APPROVED** [review]
Looks good!

**Pull request merged**